### ISSUE TITLE
Popup info: Folder contains emphasized items ?

### ISSUE THREAD

**Issue by Legends** [Microsoft/vscode#54960]
What does this mean?

**Comment by Legends** [comment]
Yeah, that's what I thought of too.
I had green dot on the right side and the folder was green too, only the emphasized items were missing.
Perhaps, my brain played a trick on me or it is something with the IDE. I will come back if it should happen again.
Thanks for the quick response!

**Comment by ali-habibzadeh** [comment]
+1

**Comment by jrieken** [comment]
> however nothing in that folder is emphasized, so I have no idea how to get rid of that. :-/

@vphantom Do you have any file/folder excludes configured? What does the SCM viewlet show?

**Comment by alexmcmillan** [comment]
I also have a folder claiming "Contains emphasized items", but all of its descendant files are green.  Reloading the window doesn't fix this.  Is there any way to determine _which_ items are "emphasized" ?

**Comment by nbjgi1** [comment]
also keen on a fix

**Comment by jrieken** [comment]
>  "Contains emphasized items", but all of its descendant files are green. Reloading the window doesn't fix this. Is there any way to determine which items are "emphasized"

All of the descendant files are meant. The message 'contains emphasized items' is just there to explain the dot, which means there are any kind decorations in the descendants

**Comment by alexmcmillan** [comment]
@jrieken If I understand you correctly, you are saying that items with the green dot (indicating they have uncommitted modifications) trigger their parent folder to become red with the "Contains Emphasized Items" message?

If so, there seems to be some kind of inconsistency; most of the time I see the "this folder containing modified files" indicated by the parent folder becoming orange and showing a dot, while the modified file shows as orange with an `M`.

The red colour and "emphasized items" message usually appears when I have a file containing syntactical errors.

Have I misunderstood you?

**Comment by alexmcmillan** [comment]
@jrieken If I understand you correctly, you are saying that files in green trigger their parent folder to become red with the "Contains Emphasized Items" message?

If so, there seems to be some kind of inconsistency; most of the time I see:

green: new, untracked files
orange: files with uncommitted modifications
red: files with errors
where the folder takes the "highest priority" colouring from the files inside it and shows a dot, while each file takes the appropriate colour and shows either "U" (untracked), "M" (modified), or a number indicating the count of syntax errors...

Have I misunderstood you?

**Comment by jrieken** [comment]
> Have I misunderstood you?

Yes. The message "Contains Emphasized Items" means that there is any kind of decoration in any of the children. The color of the dot is the one with the highest priority, usually errors or warnings. You can check the problems view for the errors and warnings current known to the editor - it should contain a problem that lives inside that subtree. Iff not check your explorer exclude settings and file a *separate* issue with steps to reproduce.

**Comment by alexmcmillan** [comment]
Thank you for your help.

There was nothing in the Problems window, but committing all files "reset" the folder structure to all-white (as expected) and the problem appears resolved.  If I come across this again I will open a new issue as you suggest.
